svgnative/src/SVGStringParser.cpp

```diff
@@ -409,6 +409,11 @@ bool ParseNumber(const std::string& numberString, float& number)
         return false;
     if (!ParseScientificNumber(pos, end, number))
         return false;
+    if (pos < end && *pos == '%')
+    {
+        number = number/100;
```

Comment:
This works for opacity where % is actually based on 1.f. But there could be other uses. I'd suggest to have a function `ParseAlphaValue()` so that it is clear why this is ok.

Also, do we use double or single precision? If the former, please add `.f` otherwise `.` at the end.